build_windows.md

```diff
@@ -16,17 +16,183 @@ Pandoc could be installed from https://pandoc.org/installing.html When performin
 "Install for all users" (to put the binaries into "Program Files" directory).
 
 ## Install WiX Toolset v3 (is preinstalled in GitHub runner)
-The latest release of the WiX Toolset can be obtained from https://wixtoolset.org/docs/wix3/.  Installing it into a clean VM might require
+The latest release of the WiX Toolset can be obtained from https://wixtoolset.org/docs/wix3/. Installing it into a clean VM might require
 an additional installation of .NET Framework 3.5 in advance
 ([instructions for adding .NET Framework 3.5 via enabling the Windows feature](https://learn.microsoft.com/en-us/dotnet/framework/install/dotnet-35-windows#enable-the-net-framework-35-in-control-panel))
 
-## Install msys2
+## Building and running without MSYS2
+
+### Install git and go
+
+To build podman you need [git](https://gitforwindows.org/) and [go](https://go.dev). In case you haven't them yet you can 
+install them opening a Windows Terminal and run the following command:
+
+```
+winget install -e GoLang.Go Git.Git
```

Comment:
winget does not come with all windows versions and flavors ... if you want to suggest people use it, then in many cases (at least mine), you have to be an admin to run it properly.

Reply:
What version of Windows are you running? My assumption was that [`winget`](https://learn.microsoft.com/en-us/windows/package-manager/winget/) is pre-installed on Windows 11 (as part of App Installer) but I may be wrong and it's only the case for latest Windows 11 or for pro flavor (the version I installed) so I have added the link.

Reply:
it is preinstalled now on Windows 10 and 11 iirc; but I do not believe it is added onto pre-existing windows installs if it was not originally installed.

Reply:
👍  I have added "...(it assumes that [winget](https://learn.microsoft.com/en-us/windows/package-manager/winget/) is installed)). This should be enough.